test/configs/control-flow-collections/analysis_options.yaml

```diff
@@ -0,0 +1,3 @@
+analyzer:
+  enable-experiment:
+    - control-flow-collections
```

Comment:
Given that the two are effectively merged at this point, you might want to also add 'spread-collections' to the list. Not a big deal if you want to wait.

Reply:
I'm hoping that these will get collapsed into one before long and we can just replace it then.